base/fabric/src/main/java/band/kessoku/lib/impl/base/fabric/KessokuBaseFabric.java

```diff
@@ -17,7 +17,6 @@
 
 import band.kessoku.lib.api.KessokuLib;
 import band.kessoku.lib.api.base.KessokuBase;
-
```

Comment:
Don't
If you don't like it, change spotless settings in build.gradle, or your changes will be overridden